apps/mark-scan/backend/src/electrical_testing/app.ts

```diff
@@ -34,7 +34,11 @@ function buildApi({
           ? { ...paperHandlerMessage, taskStatus: paperHandlerTask.getStatus() }
           : undefined,
         usbDrive: usbDriveMessage
-          ? { ...usbDriveMessage, taskStatus: usbDriveTask.getStatus() }
+          ? {
+              ...usbDriveMessage,
+              taskStatus: usbDriveTask.getStatus(),
+              underlyingDeviceStatus: await usbDrive.status(),
+            }
```

Comment:
Since we're already polling this endpoint, I decided to add the USB drive status here rather than polling for it separately